runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -3543,7 +3544,7 @@ MM_CopyForwardScheme::scanContinuationObjects(MM_EnvironmentVLHGC *env)
 					 */
 					MM_ForwardedHeader forwardedHeader(pointer, _extensions->compressObjectReferences());
 					J9Object* forwardedPtr = forwardedHeader.getForwardedObject();
```

Comment:
Move '*' next to variable (as opposed to next to type). Fix it for this method or any other method you that you've touched.